ADAPTER_CLASSES may not support appendleft

`ADAPTER_CLASSES` is defined as `Iterable`

https://github.com/scrapy/itemadapter/blob/98be1281ec11664da93f836861f81734382ce4ea/itemadapter/adapter.py#L271

However, in the docs it's suggested to use `appendleft` method which is not part of the Iterable API. 
This causes errors with tools like mypy: 

`error: "Iterable[Type[AdapterInterface]]" has no attribute "appendleft"  [attr-defined]`


Either the class type should be changed to List or other classes that may support appendleft or a different way to extend `ADAPTER_CLASSES` should be given.

--- a/README.md
+++ b/README.md
@@ -502,7 +502,7 @@ pip install zyte-common-items
 >>> from itemadapter.adapter import ItemAdapter
 >>> from zyte_common_items import Item, ZyteItemAdapter
 >>>
->>> ItemAdapter.ADAPTER_CLASSES.appendleft(ZyteItemAdapter)
+>>> ItemAdapter.ADAPTER_CLASSES = (ZyteItemAdapter, *ItemAdapter.ADAPTER_CLASSES)
 >>> item = Item()
 >>> adapter = ItemAdapter(item)
 >>> adapter

--- a/tests/test_interface.py
+++ b/tests/test_interface.py
@@ -81,10 +81,11 @@ class BaseFakeItemAdapterTest(unittest.TestCase):
     adapter_class = BaseFakeItemAdapter
 
     def setUp(self):
-        ItemAdapter.ADAPTER_CLASSES.appendleft(self.adapter_class)
+        self._adapter_classes = ItemAdapter.ADAPTER_CLASSES
+        ItemAdapter.ADAPTER_CLASSES = (self.adapter_class, *self._adapter_classes)
 
     def tearDown(self):
-        ItemAdapter.ADAPTER_CLASSES.popleft()
+        ItemAdapter.ADAPTER_CLASSES = self._adapter_classes
 
     def test_repr(self):
         item = self.item_class()

